mcodingbot/tasks/stats.py

```diff
@@ -46,16 +46,25 @@ async def update_channels(bot: Bot) -> None:
     else:
         LOGGER.warning("No view count channel to update stats for.")
 
+    # get the member count
+    if not member_channel:
+        return LOGGER.warning("No member count channel to update stats for.")
+
     guild = bot.cache.get_guild(CONFIG.mcoding_server)
     if not guild:
         return LOGGER.warning(
             "Couldn't find mCoding guild, not updating member count."
         )
-    if not member_channel:
-        return LOGGER.warning("No member count channel to update stats for.")
-    member_count = guild.member_count
-    if member_count is None:
-        return LOGGER.error("Cached guild didn't contain member count.")
+    guild_aprox_members = guild.member_count
```

Comment:
I don't think the `guild_approx_member` variable increases clarity

Reply:
It has to be stored so I can check if it's None, but still make mypy happy.

Reply:
There is also a little typo in the name.

Reply:
Mypy moment